src/redux/actions/actionCreators.js

```diff
@@ -1,37 +1,206 @@
 import * as types from './actionTypes';
 import axios from 'axios';
+import { axiosWithAuth } from '../axiosWithAuth'
+import { SPRITE_SIZE, MAP_WIDTH, MAP_HEIGHT } from '../constant'
+import { store } from '../../index';
+export const apiURL = 'https://cseu3-mud.herokuapp.com/api/';
 
-
-export const apiURL = ' https://lambda-mud-test.herokuapp.com/api/';
-
+export const logout = () => {
+  return { type: types.LOGOUT, }
+}
 
 
 export const signup = userData => dispatch => {
-    dispatch({ type: types.SIGNUP_START });
-    console.log('im thereeeeeeeees')
-    axios
+  dispatch({ type: types.SIGNUP_START });
+  console.log('im thereeeeeeeees')
+  axios
     .post(`${apiURL}registration/`, userData)
-      .then(res => {
-        localStorage.setItem('token', res.data.key)
-        dispatch({ type: types.SIGNUP_SUCCESS, payload: res.data.key });
-      })
-      .catch(err => {
-        console.log(err);
-        dispatch({ type: types.SIGNUP_FAILURE });
-      });
-  };
-
-  
+    .then(res => {
+      localStorage.setItem('token', res.data.key)
+      dispatch({ type: types.SIGNUP_SUCCESS, payload: res.data.key });
+    })
+    .catch(err => {
+      console.log(err);
+      dispatch({ type: types.SIGNUP_FAILURE });
+    });
+};
+
+
 export const login = userData => dispatch => {
-    dispatch({ type: types.LOGIN_START })
-    return axios
+  dispatch({ type: types.LOGIN_START })
+  return axios
     .post(`${apiURL}login/`, userData)
-      .then(res => {
-        localStorage.setItem('token', res.data.key)
-        dispatch({ type: types.LOGIN_SUCCESS, payload: res.data.key});
-      })
-      .catch(err => {
-        console.log(err);
-        dispatch({ type: types.LOGIN_FAILURE });
-      });
-  }
\ No newline at end of file
+    .then(res => {
+      localStorage.setItem('token', res.data.key)
+      dispatch({ type: types.LOGIN_SUCCESS, payload: res.data.key });
+    })
+    .catch(err => {
+      console.log(err);
+      dispatch({ type: types.LOGIN_FAILURE });
+    });
+}
+
+export const getRooms = userData => dispatch => {
```

Comment:
Hmm. I'm not sure about getRooms, here, as it conflicts with my getRooms.

But it's probably best just to merge it, and I can deal with the conflicts my end. :)